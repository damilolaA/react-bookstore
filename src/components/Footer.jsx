import React from 'react';

const Footer = () => {
  return (
    <section className="foot">
      <div>
        <p>&copy; 2017</p>;
      </div>
    </section>
  );
};

export default Footer;
