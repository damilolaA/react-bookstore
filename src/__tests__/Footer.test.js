import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Footer from '../components/Footer';

Enzyme.configure({ adapter: new Adapter() });

describe('Footer component', () => {

	it('should render component correctly', () => {
		const wrapper = shallow( <Footer/> );

		expect(toJson(wrapper)).toMatchSnapshot();
	});
});