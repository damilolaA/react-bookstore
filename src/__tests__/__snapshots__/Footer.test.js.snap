// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render component correctly 1`] = `
<section
  className="foot"
>
  <div>
    <p>
      Â© 2017
    </p>
  </div>
</section>
`;
